Tools/glib/api_test_runner.py

```diff
@@ -31,6 +31,7 @@
 import common
 from webkitpy.common.host import Host
 from webkitpy.common.test_expectations import TestExpectations
+from webkitpy.port.monadodriver import MonadoDriver  # noqa: E402
 from webkitpy.port.westondriver import WestonDriver
```

Comment:
Here the Wayland tests are relying on WestonDriver (which executes Weston inside Xvfb) to force the Wayland tests to always run headless. Ideally the runner should only force the WestonDriver by default, but not if the user opted to select the Wayland runner